index.html

```diff
@@ -0,0 +1,130 @@
+                <a id="about" href="./pages/about.html">About</a>
+                <a id="program" href="#">Program</a>
+                <a id="join" href="#">Join</a>
+                <a id= "sponsor" href="#">Sponsor</a>
+                <a id="news" href="#">News</a>
+                <a class="CC" href="#">CC Campaign</a>
+            </article>
+        </nav>
+        <section class="headline-container">
+            <article class="headline">
+                <h2>"Hello! Sharing world"</h2>
+                <h2>CREATIVE COMMONS<br>GLOBAL SUBMMIT<br>2023</h2>
+                <p class="intro-txt">
+                    A joyfull celebration believing in the value of openness and sharing creating 
+                    a positive change with people from over 80 countries is atking palce in Octobar, in Lagos
+                </p>
+                <h2 class="time">2023.10.15(THU) ~ 16(FRI)</h2>
+                <p>@ National Museum of Nigeria, Art Center Lagos and more</p>
+            </article>
+        </section>
+    </header>
+    <section id="main-program">
+        <h2 class="program-title wht">Main Program</h2>
+        <section id="cards">
+            <article class="card">
+                <img src="./assets/icons/home_iot_device_FILL0_wght400_GRAD0_opsz48.svg" alt="home_iot_device">
+                <h2>Home IoT</h2>
+                <p>
+                    Lorem ipsum, dolor sit amet consectetur adipisicing elit. Quos accusantium ipsam minima.
+                </p>
+            </article>
+            <article class="card">
+                <img src="./assets/icons/smart_outlet_FILL0_wght400_GRAD0_opsz48.svg" alt="smart_outlet">
+                <h2>Smart Outlet</h2>
+                <p>
+                    Lorem ipsum, dolor sit amet consectetur adipisicing elit. Quos accusantium ipsam minima.
+                </p>
+            </article>
+            <article class="card">
+                <img src="./assets/icons/assistant_on_hub_FILL0_wght400_GRAD0_opsz48.svg" alt="home_assistance">
+                <h2>Home Assistance</h2>
+                <p>
+                    Lorem ipsum, dolor sit amet consectetur adipisicing elit. Quos accusantium ipsam minima.
+                </p>
+            </article>
+            <article class="card">
+                <img src="./assets/icons/rocket_launch_FILL0_wght400_GRAD0_opsz48.svg" alt="rocket_science">
+                <h2>Rocket Science</h2>
+                <p>
+                    Lorem ipsum, dolor sit amet consectetur adipisicing elit. Quos accusantium ipsam minima.
+                </p>
+            </article>
+        </section>
+        <h2 class="Program-foot dk">SEE THE WHOLE PROGRAM</h2>
+        <h2 class="Program-foot mb">Join CC Global Summit 2023</h2>
+    </section>
+    <section id="speakers">
+
+        <h2 class="program-title">Featured Speakers</h2>
+        <section id="speaker"></section>
+        <button id="toggle-more" class="toggle-more-less mb">MORE</button>
```

Comment:
👏👏 Great job on adding the "Toggle More" button! However, there are some styling issues that need to be addressed to match the design requirements:

- [ ] The font styles for the button need to be updated to match the design.
- [ ] The border color of the button is currently missing, please add it as per the design.
- [ ] The icon for the "Toggle More" button is not present, consider adding it to enhance the button's appearance.
![image](https://github.com/sagieramos/CAPSTONE-ONE/assets/55840999/34488c97-a905-4c05-8a3f-24d540e53923)

By making these adjustments, your "Toggle More" button will look more visually appealing and in line with the overall design. Keep up the good work and happy coding! 🚀💻🌟